start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
I would prefer `#!/usr/bin/env bash` as it would use whatever is in your `$PATH` variable. On NixOS for example there is no `/bin/bash` file.